proposals/incubation/wasmedge.md

```diff
@@ -0,0 +1,127 @@
+* [_version 0.8.2_](https://github.com/WasmEdge/WasmEdge/releases/tag/0.8.2)
+* [_version 0.9.0_](https://github.com/WasmEdge/WasmEdge/releases/tag/0.9.0)
+* [_version 0.10.0_](https://github.com/WasmEdge/WasmEdge/releases/tag/0.10.0)
+* [_version 0.11.0_](https://github.com/WasmEdge/WasmEdge/releases/tag/0.11.0)
+* [_version 0.11.1_](https://github.com/WasmEdge/WasmEdge/releases/tag/0.11.1)
+* [_version 0.11.2_](https://github.com/WasmEdge/WasmEdge/releases/tag/0.11.2)
+* [_version 0.12.0_](https://github.com/WasmEdge/WasmEdge/releases/tag/0.12.0)
+* [_version 0.12.1_](https://github.com/WasmEdge/WasmEdge/releases/tag/0.12.1)
+
+With each release, WasmEdge delivers important features that enable new cloud native use cases for WasmEdge users and partners.
+
+The list below highlights some of the technical achievements the team has accomplished since joining the CNCF sandbox. 
+
+* Dynamic language support. Besides compiled languages such as Rust and C/C++, WasmEdge has added support for JavaScript, Python and PHP programs. WasmEdge supports the full JavaScript language spec as well as common JavaScript module systems, including ES6, CommonJS (CJS), and NPM modules. Python and PHP support is achieved through our collaboration with VMWare’s WasmLab. 
+* Container and K8s tooling. WasmEdge has reached OCI-compliance. That allows it to be managed and orchestrated by standard container tools, such as Docker, crun, containerd, CRI-O, K8s, Kind, OpenYurt, KubeEdge, SuperEdge.
+* Advanced networking with the WasmEdge sockets library. It supports non-blocking sockets, DNS, TLS. That allows WadmEdge to run modern HTTPS client and server applications. 
+* Support for the Rust Tokio library. It allows many Tokio-based async Rust applications, especially async networking apps, to run seamlessly on WasmEdge.
+* Connectors and drivers for commonly used microservice components, such as MySQL, PostgreSQL, Redis, Kafka, and Dapr. 
+* Support AI inference applications written in both Rust and JavaScript using Tensorflow, OpenVino, PyTorch trained models.
+* Host SDKs in Rust, Go, C, C++, Python, and Java languages.
+
+WasmEdge is the only WebAssembly runtime in the market that could run complete microservices, with support for web servers, HTTPS clients, and database connections. Through partnerships with Docker, Red Hat, and Microsoft, WasmEdge could be seamlessly integrated into the existing cloud native ecosystem.
+
+## **Incubation Stage Requirements**
+
+***Document that it is being used successfully in production by at least three independent direct adopters which, in the TOC’s judgment, are of adequate quality and scope.***
+
+The full list of companies and projects that have incorporated WasmEdge into their product offerings can be found [on our website](https://wasmedge.org/docs/contribute/users).
+
+Below is a selected list of end-user organizations and products that are using WasmEdge for business workloads.
```

Comment:
Thanks, Justin. We believe Huawei Cloud meets the "end user" definition since it is basically internal IT department providing WasmEdge function services for internal teams. They will open it up to the public later this year once they feel comfortable supporting it for the wider public.

We have listed Docker with other technology/tool providers that integrate/bundle WasmEdge in our "users" page: https://wasmedge.org/docs/contribute/users/